obs-packaging/linux-container/kata-linux-container.spec-template

```diff
@@ -82,6 +82,12 @@ BuildKernel() {
     find kata-kernel-configs -name "${kernelArch}_kata_kvm_*" -exec cp {} .config \;
 	[ -f .config ] || (echo "ERROR: cannot find the kernel config file for the ${kernelArch} architecture"; exit 1)
 
+    %if 0%{?rhel_version}
+    # RHEL in OBS has updated gcc.
```

Comment:
Should this be "RHEL needs an updated gcc in OBS"?

@lsm5, @fatherlinux, @mrguitar - could you offer any suggestions on how we can avoid doing this?